mm2src/mm2_main/tests/docker_tests_main.rs

```diff
@@ -100,6 +103,29 @@ pub fn docker_tests_runner(tests: &[&TestDescAndFn]) {
     test_main(&args, owned_tests, None);
 }
 
+fn wait_for_geth_node_ready() {
+    let mut attempts = 0;
+    loop {
+        if attempts >= 5 {
+            panic!("Failed to connect to Geth node after several attempts.");
+        }
+        match block_on(GETH_WEB3.eth().block_number().timeout(Duration::from_secs(6))) {
+            Ok(Ok(block_number)) => {
+                println!("Geth node is ready, latest block number: {:?}", block_number);
```

Comment:
We shouldn't use println in tests, we should use `log!` macro instead

Reply:
agree, fixed here https://github.com/KomodoPlatform/komodo-defi-framework/pull/2100/commits/e672485b01b309c99d79c9145716dda6058ca7ff